npm/builder.js

```diff
@@ -26,7 +26,7 @@ function Builder () {
     'node_modules/opal-runtime/src/opal.js',
     'build/asciidoctor-lib.js'
   ];
-  this.asciidoctorCoreVersion = '1.5.5';
+  this.asciidoctorCoreVersion = 'master'; // or v1.5.5 to build against a release
```

Comment:
I think that this should read from an environment variable, defaulting to a fixed version (though we can hardcode master for the time being).